### PR TITLE
Add an initial glossary

### PR DESCRIPTION
**Pull request by nspark**
Closes openshmem-org/specification#99, closes openshmem-org/specification#228

### CODE REVIEW

**Comment by nspark** [comment]
@BryantLam This is a _very last minute_ PR. I'm sorry! Please merge if you find it acceptable. Otherwise, we'll wait for 1.6.

**Comment by BryantLam** [comment]
LGTM. @jdinan: should I merge for v1.5rc2?

File: content/glossary.tex [hunk 1]
```diff
@@ -0,0 +1,48 @@
+\begin{description}
+
+\item[Blocking] An \openshmem routine for which return from a call to
+  that routine guarantees local completion for its associated local
+  buffers
```
Comment (jdinan) [inline content/glossary.tex]:
```suggestion
  buffers.
```

**Review by jdinan — APPROVED** [review]
Minor edit.

**Comment by jdinan** [comment]
@BryantLam Yes, looks good to me. Please also merge the above suggestion to add a missing period.

**Comment by jdinan** [comment]
@nspark 👏 Nice hustle. 😄

**Pull request merged**